src/qt/walletmodel.cpp

```diff
@@ -968,10 +968,11 @@ void WalletModel::sigmaMint(const CAmount& n)
             return sigma::PrivateCoin(sigmaParams, denom);
         });
 
-    auto recipients = CWallet::CreateSigmaMintRecipients(privCoins);
+    vector<CHDMint> vDMints;
+    auto recipients = CWallet::CreateSigmaMintRecipients(privCoins, vDMints);
 
     CWalletTx wtx;
-    std::string strError = pwalletMain->MintAndStoreSigma(recipients, privCoins, wtx);
+    std::string strError = pwalletMain->MintAndStoreSigma(recipients, privCoins, vDMints, wtx);
```

Comment:
not here too but please add same thing to function `updateSigmaCoins`. Actually better to use function that get all coins(both HD and non-HD mints) but I don't know the function.

Reply:
Agreed. Added to both